### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by mysqto**
Updates:
1. Remove service from `localization`
2. rearrange db by type
3. Remove `RequestIDService`

### CODE REVIEW

File: snowflake/snowflake.go [hunk 1]
```diff
@@ -168,7 +168,10 @@ func (g *Generator) setNodeIDProvider(nodeIDProvider NodeIDProvider) (err error)
 // Decompose returns a map of snowflake id parts.
 func (g *Generator) Decompose(sid uint64) (id ID) {
 	timestamp := sid >> timeShift
-	id.Timestamp = time.UnixMilli(g.Epoch.UnixMilli() + int64(timestamp*timeUnit))
+	// FIXME:
+	// after updating to go 1.17 can be replaced with
```
Comment (viettd) [inline snowflake/snowflake.go:172]: @mysqto
So this `func` is not available in `1.16`?

Comment (mysqto) [reply snowflake/snowflake.go:172]: Yes @viettd   Ref: https://golang.org/doc/go1.17

Comment (viettd) [reply snowflake/snowflake.go:172]: I see they just release `1.17.1` a few days back (fixed a few issues found on `1.17.0`), shall we do the upgrade for product repos?

Comment (mysqto) [reply snowflake/snowflake.go:172]: I think we can keep using go 1.16, and use this impl, if somebody also using this repo and not updated to go `1.17`, the code can not be complied

File: localization/localization.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+// RegisterLocales Locales
+func RegisterLocales(locales []string, localeDir string) {
+	for _, locale := range locales {
+		RegisterLocale(locale, localeDir)
+	}
+}
+
+func Localize(lang string, msgID string, data interface{}) (string, error) {
```
Comment (viettd) [inline localization/localization.go:40]: How can golint let you escape with this function without a document 😂 ?

File: localization/localization.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+		localizer, _ = localizers.Load(DefaultLocale)
+	}
+
+	return localizer.(*i18n.Localizer).Localize(&i18n.LocalizeConfig{
+		MessageID:    msgID,
+		TemplateData: data,
+	})
+}
+
+func LocalizeMulti(lang string, msgIDs []string, data interface{}) (localizedMessages map[string]string, err error) {
```
Comment (viettd) [inline localization/localization.go:54]: Same here should keep the old document if you don't. provide a better one.

File: localization/localization.go (hunk 2 above)
Comment (mysqto) [reply localization/localization.go:41]: Aha, we don't have any githook for this repo 😂

Comment (mysqto) [reply localization/localization.go:41]: updated

File: localization/localization.go (hunk 3 above)
Comment (mysqto) [reply localization/localization.go:57]: fixed

File: snowflake/snowflake.go (hunk 1 above)
Comment (viettd) [reply snowflake/snowflake.go:172]: Ok then.

**Review by viettd — APPROVED** [review]

**Pull request merged**